ee/clickhouse/views/groups.py

```diff
@@ -77,6 +77,17 @@ def get_queryset(self):
         ]
     )
     def list(self, request, *args, **kwargs):
+        """
+        List all groups of a specific group type. You must pass ?group_type_index= in the URL. To get a list of valid group types, call /api/:project_id/groups_types/
```

Comment:
This is sort of a weird API route. It might make sense to just return everything unfiltered and allow the filtering to be applied instead of erroring out when there are no filters